### PR TITLE
Remove `Module#alias_method_chain` method

### PR DESCRIPTION
**Pull request by ykzts**
[`Module#alias_method_chain`](https://github.com/rails/rails/blob/v5.0.0.beta2/activesupport/lib/active_support/core_ext/module/aliasing.rb#L26) method is a deprecated method in the Rails 5.0.0.

### CODE REVIEW

**Comment by ykzts** [comment]
oops... It has failed the test.

please wait...

File: lib/ridgepole/ext/abstract_mysql_adapter.rb [hunk 1]
```diff
@@ -1,16 +1,26 @@
...
   end
-  alias_method_chain :add_index, :alter
+end
 
-  def remove_index_with_alter!(table_name, index_name)
-    execute "ALTER TABLE #{quote_table_name(table_name)} DROP INDEX #{quote_column_name(index_name)}"
+module ActiveRecord
+  module ConnectionAdapters
+    class AbstractMysqlAdapter
+      include Ridgepole::Ext::AbstractMysqlAdapter
```
Comment (kamipo) [inline lib/ridgepole/ext/abstract_mysql_adapter.rb]: Why use `include` rather than `prepend`?

File: lib/ridgepole/schema_statements_ext.rb [hunk 2]
```diff
@@ -1,9 +1,32 @@
...
+      # Nothing to do
+    end
   end
+end
 
-  alias_method_chain :rename_table_indexes, :ignore
+module ActiveRecord
+  module ConnectionAdapters
+    class AbstractAdapter
+      include Ridgepole::SchemaStatementsExt
```
Comment (kamipo) [inline lib/ridgepole/schema_statements_ext.rb]: ditto.

**Comment by winebarrel** [comment]
I threw away the migration_comments https://github.com/winebarrel/ridgepole/pull/67
Thank you for you PR!

File: lib/ridgepole/ext/abstract_mysql_adapter.rb (hunk 1 above)
Comment (winebarrel) [inline lib/ridgepole/ext/abstract_mysql_adapter.rb]: Fixed https://github.com/winebarrel/ridgepole/pull/68
Thank you for your comment :bow:

**Pull request merged**